ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -143,7 +135,7 @@ static VALUE tracing_stop(VALUE _self) {
   gc_tracepoint = Qnil;
 
   render_event("stopped_tracing");
-  fprintf(output_file, "]\n");
+  // closing output file is handled in GvlTracing.stop code
```

Comment:
Minor: I suggest clarifying that this comment is referencing the json structure, as otherwise it can be a bit confusing without the context of this diff because the next line is an `fclose(...)` and we're commenting that we're not closing stuff ;)

```suggestion
  // closing the json syntax in the output file is handled in GvlTracing.stop code
```